<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ibmAlbaranes</class>
 <widget class="QDialog" name="ibmAlbaranes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nuevo albarán</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpAlbaran">
       <property name="title">
        <string>Datos del Albarán</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCalendarWidget" name="calendar">
            <property name="firstDayOfWeek">
             <enum>Qt::Monday</enum>
            </property>
            <property name="verticalHeaderFormat">
             <enum>QCalendarWidget::NoVerticalHeader</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="lblProveedores">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Selecciona un proveedor</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbProveedores">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="chkInventariado">
              <property name="text">
               <string>Albarán inventariado</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdAlbaran">
              <property name="text">
               <string>Añadir albarán</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="lblTotalArticulos">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Número de artículos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblTotalCoste">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Coste</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpInsercion">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Inserción de artículos</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="lblTipo">
                <property name="text">
                 <string>Tipo de productos</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblModelo">
                <property name="text">
                 <string>Modelo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblCompra">
                <property name="text">
                 <string>Coste compra</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblVenta">
                <property name="text">
                 <string>Precio recomendado</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblCantidad">
                <property name="text">
                 <string>Cantidad</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QComboBox" name="cmbTipos"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtModelo"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtCompra"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtVenta">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spnCantidad">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="cmdArticulo">
            <property name="text">
             <string>Añadir artículo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpArticulos">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Listado der artículos del albarán</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableWidget" name="tblArticulos">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tipo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Modelo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Precio coste</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Precio recomendado</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Facturado</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionArticuloBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar artículo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/gnuoptics.qrc"/>
 </resources>
 <connections/>
</ui>
